Sources/KlaviyoForms/InAppForms/IAFWebViewModel.swift

```diff
@@ -73,6 +73,15 @@ class IAFWebViewModel: KlaviyoWebViewModeling {
         return WKUserScript(source: sdkVersionScript, injectionTime: .atDocumentEnd, forMainFrameOnly: true)
     }
 
+    @MainActor
+    private var dataEnvironmentWKScript: WKUserScript {
+        guard let formsEnv = environment.formsDataEnvironment() else {
+            return WKUserScript(source: "", injectionTime: .atDocumentEnd, forMainFrameOnly: true)
+        }
+        let sdkVersionScript = "document.head.setAttribute('data-forms-data-environment', '\(formsEnv)');"
+        return WKUserScript(source: sdkVersionScript, injectionTime: .atDocumentEnd, forMainFrameOnly: true)
+    }
+
```

Comment:
Two kinda nit picks:
1. Does creating a WKUserScript object have to run on the main thread?
2. Wouldn't it be cleaner to just not inject anything, rather than inject an empty string script?